src/controllers/AnnotationModeController.js

```diff
@@ -587,7 +587,9 @@ class AnnotationModeController extends EventEmitter {
      * @return {void}
      */
     renderPage(pageNum: string) {
-        const pageEl = this.annotatedElement.querySelector(`[data-page-number="${pageNum}"]`);
+        const pageEl = this.isMobile
```

Comment:
maybe this could be turned into a utility method? seeing it repeated a few times